tests/integration_tests/performance/configs/snap_restore_test_config.json

```diff
@@ -0,0 +1,538 @@
+                      "delta_percentage": 20
+                    },
+                    "P90": {
+                      "target": 6.529,
+                      "delta_percentage": 20
+                    }
+                  },
+                  "1vcpu_256mb": {
+                    "P50": {
+                      "target": 4.869,
+                      "delta_percentage": 20
+                    },
+                    "P90": {
+                      "target": 5.142,
+                      "delta_percentage": 20
+                    }
+                  },
+                  "1vcpu_512mb": {
+                    "P50": {
+                      "target": 5.36,
+                      "delta_percentage": 20
+                    },
+                    "P90": {
+                      "target": 6.525,
+                      "delta_percentage": 20
+                    }
+                  },
+                  "1vcpu_1024mb": {
+                    "P50": {
+                      "target": 6.29,
+                      "delta_percentage": 20
+                    },
+                    "P90": {
+                      "target": 7.451,
+                      "delta_percentage": 20
+                    }
+                  },
+                  "1vcpu_2048mb": {
+                    "P50": {
+                      "target": 10.04,
+                      "delta_percentage": 20
+                    },
+                    "P90": {
+                      "target": 11.112,
+                      "delta_percentage": 20
+                    }
+                  },
+                  "1vcpu_4096mb": {
+                    "P50": {
+                      "target": 12.58,
+                      "delta_percentage": 20
+                    },
+                    "P90": {
+                      "target": 13.725,
+                      "delta_percentage": 20
+                    }
+                  },
+                  "1vcpu_8192mb": {
+                    "P50": {
+                      "target": 21.008,
+                      "delta_percentage": 20
+                    },
+                    "P90": {
+                      "target": 22.144,
+                      "delta_percentage": 20
+                    }
+                  },
+                  "1vcpu_16384mb": {
+                    "P50": {
+                      "target": 36.616,
+                      "delta_percentage": 20
+                    },
+                    "P90": {
+                      "target": 38.455,
+                      "delta_percentage": 20
+                    }
+                  },
+                  "1vcpu_32768mb": {
+                    "P50": {
+                      "target": 67.457,
+                      "delta_percentage": 20
+                    },
+                    "P90": {
+                      "target": 69.69,
+                      "delta_percentage": 20
+                    }
+                  },
+                  "2net_dev": {
```

Comment:
What about this identifier? Would we find useful the information about the vm config?